pkg/operators/wasm/wasm.go

```diff
@@ -76,7 +78,10 @@ func (w *wasmOperator) InstantiateImageOperator(
 		createdMap:  map[uint32]struct{}{},
 	}
 
-	if err := instance.init(gadgetCtx, target, desc); err != nil {
+	// Refresh the cache to avoid using stale data
```

Comment:
Could you please describe a bit more when stale data could be used?

Also, better to move this to init()?